processor/casttosumprocessor/README.md

```diff
@@ -0,0 +1,27 @@
+# Cast To Sum Processor
```

Comment:
Consider putting this inside `agentmetricsprocessor`, which is already our catch-all processor for magic/bespoke processor behavior. It would reduce the amount of boilerplate you have here.

Reply:
Isn't `agentmetricsprocessor` designed to operate on the metrics produced by `hostmetricsreceiver`? It seems a bit hacky to also apply it to the IIS metrics (which is what this is designed for). Besides, having a separate processor makes it a lot easier to just blow away when the upstream receiver produces the correct metrics natively.